plugins/importexport/doaj/classes/DOAJExportDom.inc.php

```diff
@@ -20,22 +20,35 @@ class DOAJExportDom {
 	 * Generate the export DOM tree for a given journal.
 	 * @param $doc object DOM object
 	 * @param $journal object Journal to export
+	 * @param $selectedObjects array
 	 */
-	function &generateJournalDom(&$doc, &$journal) {
-		$issueDao =& DAORegistry::getDAO('IssueDAO');
-		$sectionDao =& DAORegistry::getDAO('SectionDAO');
-		$pubArticleDao =& DAORegistry::getDAO('PublishedArticleDAO');
+	function generateJournalDom($doc, $journal, $selectedObjects) {
```

Comment:
I don't mind if we start changing these now -- but the reason the references are used so heavily is that PHP4 treats objects very differently than PHP5, and the references bridged that gap. I think our user community has (finally ... finally) moved beyond PHP4, so I'm content to let our compatibility slip.